HTML/mockup-ellipsis/style.css

```diff
@@ -0,0 +1,458 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    /* 1rem = 10px  10px/16px = 0.625 = 62.5% */
+    font-size: 62.5%;
+}
+
+@media only screen and (max-width: 1100px) {
+    html {
+        font-size: 57%;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    html {
+        font-size: 50%;
+    }
+}
+
+/***** HEADER *****/
+
+header {
+    padding-top: 13%;
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    background-image: url(img/layer-8@3x-min.png);
+    background-size: 100% 100%;
+}
+
+@media only screen and (max-width: 1100px) {
+    header {
+        padding-top: 10%;
+        height: 70vh;
+        background-size: cover;
+        background-position: center;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    header {
+        padding-top: 7%;
+        height: 50vh;
+    }
+}
+
+.header__container {
+    width: 50%;
+    position: relative;
+    text-align: center;
+}
+
+@media only screen and (max-width: 1100px) {
+    .header__container {
+        width: 60%;
+    }
+}
+
+@media only screen and (max-width: 800px) {
+    .header__container {
+        width: 80%;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .header__container {
+        width: 90%;
+    }
+}
+
+picture {
+    display: block;
+}
+
+.header__logo {
+    width: 6.5rem;
+    height: 6.9rem;
+}
+
+.header__title {
+    margin-top: 8%;
+    font-family: Lato;
+    font-size: 5.4rem;
+    font-weight: bold;
+    letter-spacing: 8.6px;
+    text-align: center;
+    color: #ffffff;
+    text-transform: uppercase;
+}
+
+@media only screen and (max-width: 1100px) {
+    .header__title {
+        font-size: 4.5rem;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .header__title {
+        font-size: 3.5rem;
+    }
+}
+
+.header__paragraph {
+    margin-top: 3%;
+    font-family: Lato;
+    font-size: 1.5rem;
+    line-height: 1.93;
+    letter-spacing: 0.9px;
+    text-align: center;
+    color: #ffffff;
+    text-transform: uppercase;
+}
+
+@media only screen and (max-width: 500px) {
+    .header__paragraph {
+        font-size: 1.2rem;
+    }
+}
+
+.header__button {
+    width: 2.3rem;
+    height: 3.7rem;
```

Comment:
the image doesn't fit to its container. It may lead unexpected overlapping. That's why I don't recommend to set both width and height
![image](https://user-images.githubusercontent.com/11594423/59269105-82203f80-8c4e-11e9-8208-4b7a2739eff4.png)